Proto/Collector/Trace/V1/TraceServiceClient.php

```diff
@@ -0,0 +1,53 @@
+<?php
+// GENERATED CODE -- DO NOT EDIT!
+
+// Original file comments:
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+namespace Opentelemetry\Proto\Collector\Trace\V1;
+
+/**
+ * Service that can be used to push spans between one Application instrumented with
+ * OpenTelemetry and an collector, or between an collector and a central collector (in this
+ * case spans are sent/received to/from multiple Applications).
+ */
+class TraceServiceClient extends \Grpc\BaseStub {
```

Comment:
We talked about this in DM; it might make sense to skip phan checks on the proto repo (should we decide we actually want to commit this folder to the repo)